packages/ckeditor5-package-generator/tests/utils/choose-package-manager.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * @license Copyright (c) 2020-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+const { expect } = require( 'chai' );
+const mockery = require( 'mockery' );
+const sinon = require( 'sinon' );
+
+describe( 'lib/utils/choose-package-manager', () => {
+	let stubs, choosePackageManager;
+
+	beforeEach( () => {
+		mockery.enable( {
+			useCleanCache: true,
+			warnOnReplace: false,
+			warnOnUnregistered: false
+		} );
+
+		stubs = {
+			inquirer: {
+				prompt: sinon.stub()
+			},
+			isYarnInstalled: sinon.stub()
+		};
+
+		mockery.registerMock( './is-yarn-installed', stubs.isYarnInstalled );
+		mockery.registerMock( 'inquirer', stubs.inquirer );
+
+		choosePackageManager = require( '../../lib/utils/choose-package-manager' );
+	} );
+
+	afterEach( () => {
+		mockery.deregisterAll();
+		mockery.disable();
+		sinon.restore();
+	} );
+
+	it( 'should return npm when npm argument is true and yarn is installed', async () => {
+		stubs.isYarnInstalled.returns( true );
+
+		const result = await choosePackageManager( { useNpm: true } );
+
+		expect( result ).to.equal( 'npm' );
+	} );
+
+	it( 'should return npm when arguments are true and yarn is installed', async () => {
```

Comment:
If both `--use-npm` and `--use-yarn` flags are set and `yarn` is installed, I think we should ask user.